docs/taskruns.md

```diff
@@ -827,16 +827,46 @@ TaskRuns can be halted on failure for troubleshooting by providing the following
 ```yaml
 spec:
   debug:
-    breakpoint: ["onFailure"]
+    breakpoints: 
+      onFailure: true
 ```
 
-Upon failure of a step, the TaskRun Pod execution is halted. If this TaskRun Pod continues to run without any lifecycle
-change done by the user (running the debug-continue or debug-fail-continue script) the TaskRun would be subject to
+### Breakpoint before step
+
+If you want to set a breakpoint before the step is executed, you can add the step name to the `beforeSteps` field in the following way:
+
+```yaml
+spec:
+  debug:
+    breakpoints:
+      beforeSteps: 
+        - {{ stepName }}
+```
+
+### Breakpoint after step
+
+`after step` will take effect when the commands are executed without error, 
```

Comment:
TBH, I don't understand the use case for this, and I don't see it in the TEP. What's the purpose of a breakpoint that only takes effect when the steps execute without error?

Reply:
hi @lbernick, the reason for this is that it will conflict with onfailure-breakpoint. When an error occurs during step execution, onfailure-breakpoint should handle this.

Reply:
I think afterStep-breakpoint and onfailure-breakpoint should focus on their respective functions, because when onfailure-breakpoint is triggered, afterStep-breakpoint is meaningless